awx/ui_next/src/screens/InstanceGroup/shared/InstanceGroupForm.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
```

Comment:
@nixocio Since IGs don't have an add subscreen, this component isn't actually shared. I think we should move this file to `InstanceGroup/InstanceGroupEdit` and delete the `shared` folder.

Reply:
`InstanceGroupForm` is shared between the `InstanceGroupAdd` and `InstanceGroupEdit` even in this case, is your suggestion still valid? I was trying to follow the same pattern that already in code basis. But it seems to be valid just in cases we have a shared `subscreen`. I will evaluate this change.

Reply:
@nixocio Whoops, you're right. Looks like IGs _do_ have an add form. Never mind! It is fine as is.